src/test/java/com/kodilla/ecommercee/domain/GroupTestSuite.java

```diff
@@ -0,0 +1,101 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.GroupRepository;
+import com.kodilla.ecommercee.repository.ProductRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import javax.transaction.Transactional;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+
+@SpringBootTest
+@Transactional
+class GroupTestSuite {
+    @Autowired
+    private GroupRepository groupRepository;
+    @Autowired
+    ProductRepository productRepository;
+
+
+    private Group shirt;
+    private Group shorts;
+    private List<Product> allProducts;
+
+
+    @BeforeEach
+    public void init() {
+        groupRepository.deleteAll();
+    }
+
+    @Test
+    public void testCreateNewGroup() {
+        //Given
+        shirt = new Group();
+        shorts = new Group();
+        shirt = groupRepository.save(shirt);
+        shorts = groupRepository.save(shorts);
+        Group newGroup = new Group();
+        //When
+        groupRepository.save(newGroup);
+        //Then
+        List<Group> allGroups = (List<Group>) groupRepository.findAll();
+        assertEquals(3, allGroups.size());
+        //cleanup
```

Comment:
The cleanup is unnecessary when we test on H2 database

Reply:
I also had CleanUp in my tests. I added this section so that there would be no problems with the tests if someone ever changed the database. It's not mandatory but it's good to have it.